src/background/sidebar-injector.js

```diff
@@ -189,6 +187,7 @@ export function SidebarInjector() {
 
     const canInject = await canInjectScript(tab.url);
     if (canInject) {
+      /** @type {{type: string}[]}} */
```

Comment:
```suggestion
      /** @type {Array<{ type: string } | null>} */
```

Entries within this result could be `null`. I changed from `T[]` to `Array<T>` syntax here because this introduced a union.